src/invert/laplace/impls/iterative_parallel_tri/iterative_parallel_tri.hxx

```diff
@@ -195,17 +214,23 @@ private:
   int nproc;
 
   /// Array recording whether a kz mode is converged
-  Array<bool> converged ;
+  Array<bool> converged;
 
   /// Error interpolated onto the grid one finer than current grid
   Matrix<dcomplex> fine_error;
 
-  /// Number of local x, y, z points
-  int ncx, ny;
-
   /// First and last interior points xstart, xend
   int xs, xe;
 
+  bool isGlobalFlagSet(int flag) {
+    return (global_flags & flag) != 0;
+  }
+  bool isInnerBoundaryFlagSet(int flag) {
+    return (inner_boundary_flags & flag) != 0;
+  }
+  bool isOuterBoundaryFlagSet(int flag) {
+    return (outer_boundary_flags & flag) != 0;
+  }
```

Comment:
These helper functions might actually be more generally useful, and could maybe be moved to `Laplace` base class? Could then make the flag members `private` rather than `protected`

And I've just noticed these methods should all be `const` too